scripts/genappimage.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+########################################################################
+# Package the baries built on Travis-CI as an AppImage
+# By Simon Peter 2016
+# For more information, see http://appimage.org/
+########################################################################
+
+if [ -z "$ARCH" ]; then
+  export ARCH="$(arch)"
+fi
+
+APP=NVim
+LOWERAPP="${APP,,}"
```

Comment:
No need to use weird bash features here, can simply hardcode a string. Though I guess this needs to be nvim and not tolower($APP).